selenium/src/main/java/ru/tinkoff/qa/neptune/selenium/functions/browser/proxy/BrowserProxyCriteria.java

```diff
@@ -0,0 +1,539 @@
+package ru.tinkoff.qa.neptune.selenium.functions.browser.proxy;
+
+import com.browserup.harreader.model.*;
+import ru.tinkoff.qa.neptune.core.api.steps.Criteria;
+
+import java.net.http.HttpClient;
+import java.util.Date;
+import java.util.List;
+import java.util.Objects;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import static java.lang.String.format;
+import static java.net.http.HttpClient.Version.HTTP_1_1;
+import static java.net.http.HttpClient.Version.HTTP_2;
+import static java.util.Objects.nonNull;
+import static java.util.Optional.ofNullable;
+import static java.util.regex.Pattern.compile;
+import static org.apache.commons.lang3.StringUtils.isNotBlank;
+import static ru.tinkoff.qa.neptune.core.api.steps.Criteria.condition;
+
+public final class BrowserProxyCriteria {
+
+    private BrowserProxyCriteria() {
+        super();
+    }
+
+    /**
+     * Checks the url of request.
+     *
+     * @param url is the url request is supposed to have
+     * @return criteria that checks HAR entry
+     */
+    public static Criteria<HarEntry> requestUrl(String url) {
```

Comment:
I think it would be better if every static criteria-methad started with `recored` or `proxied`.
```java public static Criteria<HarEntry> recordedRequestUrl(String url) {```

Some confusion is possible. See `ru.tinkoff.qa.neptune.http.api.response.ResponseCriteria`